lib/Agoneum/account/account.ex

```diff
@@ -63,7 +63,20 @@ defmodule Agoneum.Account do
   """
   def create_user(attrs \\ %{}) do
```

Comment:
Scratch that, its `Account.create_user`. Should we maybe call this `Accounts` like in http://swanros.com/phoenix-1-3-is-pure-love-for-api-development/ ?